components/docker-auth/pkg/auth/authz_handler.go

```diff
@@ -19,24 +19,27 @@
 package auth
 
 import (
+	"fmt"
+
 	"database/sql"
-	"log"
+
+	"github.com/cesanta/docker_auth/auth_server/api"
+	"go.uber.org/zap"
 
 	"github.com/cellery-io/cellery-hub/components/docker-auth/pkg/extension"
 )
 
-func Authorization(dbConn *sql.DB, accessToken string, execId string) int {
-	log.Printf("[%s] Authorization logic handler reached and access will be validated\n", execId)
-	isValid, err := extension.ValidateAccess(dbConn, accessToken, execId)
+func Authorization(dbConn *sql.DB, ai *api.AuthRequestInfo, logger *zap.SugaredLogger, execId string) (bool, error) {
```

Comment:
Is this function coming from an interface ? If this is defined by you can we change the name of the function >

Reply:
Fixed by #159